.github/workflows/test.yml

```diff
@@ -0,0 +1,63 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - staging
+  pull_request:
+    branches:
+      - staging
+
+env:
+  NODE_ENV: test
+
+jobs:
+  client-tests:
+    name: Client Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
```

Comment:
Usually the matrix makes sense if we were some web library trying to be backwards compatible, but since our site will typically only use one version of node, it's safe to remove the matrix in favor of just one node version, in this case, 12.x.

Reply:
@Gocnak I set it like this because that's how it appears in the existing `AngularBuild` workflow in this repo, I supposed it was because the site would be moving towards multiple versions soon so left it in. Should it be removed from one or both or none?

Reply:
They can be removed from both! I think it was left in the AngularBuild one by accident, we'll only have one node target yep.